Modelica/Mechanics/Translational.mo

```diff
@@ -1697,6 +1697,64 @@ An eddy current brake reduces the speed of a moving mass. Kinetic energy is conv
 </html>"));
     end EddyCurrentBrake;
 
+    model Vehicle1d "1-dimensional vehicle with driving resistances"
```

Comment:
* Rename Vehicle1d to Vehicle1D, since we also have Rotational3DEffects for example.
* Change "1-dimensional vehicle with driving resistances" to "One-dimensional vehicle with driving resistances"

Reply:
done.